### PR TITLE
HRINT-1267 - licensing enhancements

### PR DESCRIPTION
**Pull request by grisha-kotler**

### CODE REVIEW

File: src/Raven.Client/Properties/VersionInfo.cs [hunk 1]
```diff
@@ -4,7 +4,7 @@
 using Raven.Client.Extensions;
 using Raven.Client.Properties;
 
-[assembly: RavenVersion(Build = "42", CommitHash = "a377982", Version = "4.2", FullVersion = "4.2.104-custom-42")]
+[assembly: RavenVersion(Build = "42", CommitHash = "a377982", Version = "4.2", FullVersion = "4.2.104-custom-42", EffectiveReleaseDateString = "2019-05-21")]
```
Comment (ayende) [inline src/Raven.Client/Properties/VersionInfo.cs]: Isn't this automatically generated?

**Review by ayende — APPROVED** [review]

File: src/Raven.Client/Properties/VersionInfo.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Client/Properties/VersionInfo.cs]: It is

File: src/Raven.Client/Properties/VersionInfo.cs [hunk 2]
```diff
@@ -4,7 +4,7 @@
 using Raven.Client.Extensions;
 using Raven.Client.Properties;
 
-[assembly: RavenVersion(Build = "42", CommitHash = "a377982", Version = "4.2", FullVersion = "4.2.104-custom-42", EffectiveReleaseDateString = "2019-05-21")]
+[assembly: RavenVersion(Build = "42", CommitHash = "a377982", Version = "4.2", FullVersion = "4.2.104-custom-42", ReleaseDateString = "2020-07-02")]
```
Comment (ayende) [inline src/Raven.Client/Properties/VersionInfo.cs:7]: The one thing that pops to mind, now this is on each release, what updates this? Do we need to remember doing that?

Comment (ppekrol) [reply src/Raven.Client/Properties/VersionInfo.cs:7]: @gregolsky will handle that on CI

**Review by ppekrol — APPROVED** [review]

**Pull request merged**